### PR TITLE
Restructure org config dict to use common and migrate api token to or…

### PR DESCRIPTION
**Pull request by norkans7**
…g config

### CODE REVIEW

File: dash/orgs/migrations/0019_resctructure_org_config.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                old_config = dict()
+            else:
+                old_config = json.loads(self.config)
+
+            if "common" in old_config and "rapidpro" in old_config:
+                print("Skipped org(%d), it looks like already migrated" % org.id)
+                continue
+
+            new_config = {"common": old_config, "rapidro": {"api_token": org.api_token}}
+            self.config = json.dumps(new_config)
```
Comment (rowanseymour) [inline dash/orgs/migrations/0019_resctructure_org_config.py]: side note: let's update this to real JSON field at some point

File: dash/orgs/migrations/0019_resctructure_org_config.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
```
Comment (rowanseymour) [inline dash/orgs/migrations/0019_resctructure_org_config.py]: typo in name of this file

File: dash/orgs/models.py [hunk 3]
```diff
@@ -150,7 +153,9 @@ def get_temba_client(self, api_version=1):
 
         client_cls = TembaClient1 if api_version == 1 else TembaClient2
 
-        return client_cls(host, self.api_token, user_agent=agent)
+        api_token = self.get_config('api_token', top_key="rapidpro")
+
+        return client_cls(host, api_token, user_agent=agent)
```
Comment (rowanseymour) [inline dash/orgs/models.py]: side note: do we need to still support version 1?

Comment (norkans7) [reply dash/orgs/models.py]: No I think

**Pull request merged**